receiver/opencensusreceiver/ocmetrics/opencensus.go

```diff
@@ -130,29 +136,45 @@ func processReceivedMetrics(ni *commonpb.Node, resource *resourcepb.Resource, me
 }
 
 func (ocr *Receiver) batchMetricExporting(longLivedRPCCtx context.Context, payload interface{}) {
-	mds := payload.([]*consumerdata.MetricsData)
-	if len(mds) == 0 {
-		return
-	}
+	ctx, span := obsreport.StartMetricsReceiveOp(
+		context.Background(),
+		ocr.instanceName,
+		receiverTransport)
 
-	// Trace this method
-	ctx, span := trace.StartSpan(context.Background(), "OpenCensusMetricsReceiver.Export")
-	defer span.End()
+	// If the starting RPC has a parent span, then add it as a parent link.
+	obsreport.SetParentLink(longLivedRPCCtx, span)
 
-	// TODO: (@odeke-em) investigate if it is necessary
-	// to group nodes with their respective metrics during
-	// bundledMetrics list unfurling then send metrics grouped per node
+	mds := payload.([]*consumerdata.MetricsData)
+	numTimeSeries := 0
+	numPoints := 0
+	var consumerErr error
+	if len(mds) != 0 {
+		for _, md := range mds {
+			if md == nil || len(md.Metrics) == 0 {
+				continue
+			}
 
-	// If the starting RPC has a parent span, then add it as a parent link.
-	observability.SetParentLink(longLivedRPCCtx, span)
+			// Legacy count number of time series, there was no legacy metric
+			// here before but count it anyway for consistency
```

Comment:
I am confused by the second part of the comment.

Reply:
I updated the comment but considering removing it feels more like a comment for the review itself, I don't think it will age well.

Reply:
Done.

Reply:
I think removing makes sense.